### ISSUE TITLE
If multiple Accept headers are provided with different priorities, wiremock accepts only the first one

### ISSUE THREAD

**Issue by nikolavp** [tomakehurst/wiremock#334]
I am trying to a test a service that accepts different content types. But if I start wiremock infront, I can no longer rely on the priority (q) parameter for those because wiremock only reads the first accept occurance and stops there. For example if I have a request with the following accept headers:

```
Accept: text/csv;q=0.8,
Accept: application/sparql-results+json;q=0.8,
Accept: application/json;q=0.8,
Accept: application/sparql-results+xml;q=0.8,
Accept: application/xml;q=0.8, 
Accept: text/tab-separated-values;q=0.8,
Accept: application/x-binary-rdf-results-table
```

I am getting the response in csv through wiremock but if I hit the server directly, I am getting the response in *application/x-binary-rdf-results-table* which is to be expected. In the logs I can see that wiremock doesn't send the additional headers but just the first occurance of the Accept header, so the server is only getting 
```
Accept: text/csv;q=0.8
```

**Comment by tomakehurst** [comment]
This looks like a bug in the proxy. Thanks for pointing this out.

Unfortunately, other than fixing it I can't think of a workaround at the moment.

**Comment by tomakehurst** [comment]
Fixed in 2.0.8-beta
